modules/osdk-updating-v1101-to-v1160.adoc

```diff
@@ -0,0 +1,195 @@
+
+See link:https://docs.openshift.com/container-platform/4.9/release_notes/ocp-4-9-release-notes.html#ocp-4-9-osdk-k8s-api-bundle-validate[Validating bundle manifests for APIs removed from Kubernetes 1.22] and link:https://docs.openshift.com/container-platform/4.9/release_notes/ocp-4-9-release-notes.html#ocp-4-9-removed-kube-1-22-apis[Beta APIs removed from Kubernetes 1.22] for more information about changes introduced in Kubernetes 1.22.
+====
+
+.Prerequisites
+
+- Operator SDK {osdk_ver} installed
+- An Operator project created or maintained with Operator SDK {osdk_ver_n1}
+
+.Procedure
+
+. Add the `protocol` field in the `config/default/manager_auth_proxy_patch.yaml` and `config/rbac/auth_proxy_service.yaml` files:
++
+[source,diff]
+----
+...
+ ports:
+ - containerPort: 8443
++  protocol: TCP
+   name: https
+----
+
+. Make the following changes to the `config/manager/manager.yaml` file:
+
+.. Increase the CPU and memory resource limits:
++
+[source,diff]
+----
+resources:
+  limits:
+-     cpu: 100m
+-     memory: 30Mi
++     cpu: 200m
++     memory: 100Mi
+----
+
+.. Add an annotation to specify the default container manager:
++
+[source,yaml]
+----
+...
+template:
+  metadata:
+    annotations:
+      kubectl.kubernetes.io/default-container: manager
+...
+----
+
+. Add `PHONY` targets to all of the targets in your `Makefile` file.
+
+. For Go-based Operator projects, make the following changes:
+
+.. Install the `setup-envtest` binary.
+
+.. Change your `go.mod` file to update the dependencies:
++
+[source,golang]
+----
+k8s.io/api v0.22.1
+k8s.io/apimachinery v0.22.1
+k8s.io/client-go v0.22.1
+sigs.k8s.io/controller-runtime v0.10.0
+----
+
+.. Run the `go mod tidy` command to download the dependencies:
++
+[source,terminal]
+----
+$ go mod tidy
+----
+
+.. Make the following changes to your `Makefile` file:
++
+[source,diff]
+----
+...
+- ENVTEST_K8S_VERSION = 1.21`
++ ENVTEST_K8S_VERSION = 1.22`
+
+  test: manifests generate fmt vet envtest ## Run tests.
+-   go test ./... -coverprofile cover.out
++   KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
+...
+
+- $(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
++ $(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+...
+
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+- CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
+...
+- admissionReviewVersions={v1,v1beta1}
++ admissionReviewVersions=v1
+...
+
++ ifndef ignore-not-found
++   ignore-not-found = false
++ endif
+
+##@ Deployment
+...
+- sh kubectl delete -f -
++ sh kubectl delete --ignore-not-found=$(ignore-not-found) -f -
+----
+
+.. Run the `make manifest` command to generate your manifests with the updated version of Kubernetes:
++
+[source,terminal]
+----
+$ make manifest
```

Comment:
Should this have an oc ahead of the command? Such as: `$ oc make manifest`

Reply:
In this case, `make manifest` is part of Go, so no `oc` command necessary.